test/cypress/integration/settings/your-psp-stripe-tasklist.cy.test.js

```diff
@@ -110,4 +118,66 @@ describe('Your PSP Stripe page', () => {
     cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
     cy.get('span').contains('Government entity document').should('have.attr', 'href', `/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}/government-entity-document`)
   })
+
+  describe('Bank details task', () => {
+    it('should click bank details task and display bank details page correctly', () => {
+      setupYourPspStubs({
+        gatewayAccountId,
+        bankAccount: false,
+        director: false,
+        vatNumber: false,
+        companyNumber: false,
+        responsiblePerson: false,
+        organisationDetails: false,
+        governmentEntityDocument: false
+      })
+
+      cy.setEncryptedCookies(userExternalId)
+      cy.visit(`/account/${gatewayAccountExternalId}/your-psp/${credentialExternalId}`)
+      cy.get('span').contains('Bank Details').click()
+      cy.get('#bank-details-form').should('exist')
+        .within(() => {
+          cy.get('input#account-number').should('exist')
+          cy.get('input#sort-code').should('exist')
+          cy.get('button').should('exist')
+          cy.get('button').should('contain', 'Save and continue')
+        })
+    })
+
+    it('should redirect back to the task List when valid bank details submitted', () => {
+      setupYourPspStubs({
+        gatewayAccountId,
+        bankAccount: false,
+        director: false,
+        vatNumber: false,
+        companyNumber: false,
+        responsiblePerson: false,
+        organisationDetails: false,
+        governmentEntityDocument: false
+      })
+
+      cy.get('input#account-number[name="account-number"]').type(accountNumber)
+      cy.get('input#sort-code[name="sort-code"]').type(sortCode)
+      cy.get('#bank-details-form > button').click()
```

Comment:
I would move this line into the next test.
Then you don't have to run cy.visit.

Reply:
We can ignore this - as after speaking to the team about this, realised this is a constraint with how we do Cypress tests at the moment.  In the future, we will improve this, by splitting out the tests.